srcs/layouts/cyrl_jcuken_ru.xml

```diff
@@ -2,41 +2,41 @@
 <keyboard name="ЙЦУКЕН (Русский)" script="cyrillic">
```

Comment:
@carrot-cookie and @Thunder-Squirrel, you edited this layout in the past, what do you think of these changes ?